wahlordnung.md

```diff
@@ -64,54 +64,60 @@ Diese Wahlordnung gilt für alle Wahlen zum Fachschaftsrat IT-Systems Engineerin
+(1) Gibt es für die Wahl für das studentische Mitglied im Prüfungsausschuss keine Kandidatur und hat bis zum Beginn der Stimmabgabe keine wahlberechtigte Person gegenüber der Fachschaft zu verstehen gegeben, für das Amt zur Verfügung zu stehen, so findet abweichend von § 2 (3) keine Wahl für das studentische Mitglied im Prüfungsausschuss durch die gesamte Fachschaft statt.
 
 (2) Das Amt muss in diesem Fall von einem Mitglied des Fachschaftsrates übernommen werden.
 
-(3) Die Wahl des studentischen Mitglieds im Prüfungsausschuss findet in diesem Fall während der konstituierenden Sitzung des neuen Fachschaftsrates statt. Die §§ 4 bis 12 finden dabei keine Anwendung. Das Verfahren entspricht dem der in §5a der Satzung der Fachschaft IT-Systems Engineering genannten Ämter.
+(3) Die Wahl des studentischen Mitglieds im Prüfungsausschuss findet in diesem Fall analog zur in § 3 (4) geregelten Wahl des vom Fachschaftsrat gewählten studentischen Mitglieds in der Studienkommission statt.
 
 
 ## § 9 Wahldurchführung
 
-(1) Jedes Mitglied der Fachschaft hat ebenso viele Stimmen, wie Mitglieder des jeweiligen Gremiums zu wählen sind.
+(1) Jedes Mitglied der Fachschaft hat ebenso viele Stimmen, wie Mitglieder des jeweiligen Gremiums zu wählen sind. Stimmenhäufung ist nicht zulässig.
 
-(2) Dem Wähler steht es offen, seine Stimmen auf die Kandidaten zu verteilen. Stimmenhäufung ist nicht zulässig.
+(2) Wählende sind nicht an die Wahlvorschläge gebunden.
 
-(3) Der Wähler ist nicht an die Kandidaten gebunden.
+(3) Wahlberechtigten wird ab der Bekanntgabe der Wahlvorschläge die Möglichkeit gegeben, ihre Stimme per E-Mail abzugeben. Dieses Verfahren muss einer Briefwahl insofern entsprechen, als abgegebene Stimmen der wählenden Person nicht zugeordnet werden dürfen. Dafür ist die Stimmabgabe für jedes zu wählende Gremium einzeln in getrennt von der E-Mail auszuwertenden Anhängen zu tätigen. Die Abgabe der Stimmen per E-Mail muss vom HPI-Konto erfolgen.
 
-(4) Wahlberechtigten wird ab der Bekanntgabe der Kandidaten die Möglichkeit gegeben, ihre Stimme per E-Mail abzugeben. Dieses Verfahren muss einer Briefwahl insofern entsprechen, als abgegebene Stimmen dem Wähler nicht zugeordnet werden dürfen. Dafür ist die Stimmabgabe in einem getrennt von der E-Mail auszuwertenden Anhang zu tätigen. Die Abgabe der Stimmen per E-Mail muss vom HPI-Konto des Wählers erfolgen.
+(4) Wenn eine Person sowohl per E-Mail als auch per Wahlzettel abgestimmt hat, verfällt die Stimmabgabe aus der E-Mail.
 
-(5) Sofern ein Wähler sowohl per E-Mail als auch per Wahlzettel abgestimmt hat, verfällt die Stimmabgabe aus der E-Mail.
+(5) Bei mehreren Personen gleichen Namens wird für die Stimmabgabe der in der HPI-Domäne geführte Name mit nachgestellter Zahl verwendet. Der Wahlausschuss weist in der Wahlausschreibung und zur Eröffnung der Wahl darauf hin.
```

Comment:
weil in der ausschreibung und der durchführung darauf geachtet werden muss. damit wollte ich verhindern, das es an einer stelle vergessen wird.